examples/models/legends.py

```diff
@@ -1,3 +1,11 @@
+'''This example shows different ways to place legends on a graph using LegendLocation. 
+The legend can be placed at defined axes, fixed locations, name locations and side panels. 
```

Comment:
This example only demonstrates placing legends in the named locations included in the enumeration. You can just omit this sentence.

```suggestion

```